.travis.yml

```diff
@@ -1,37 +1,73 @@
-language: python
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
-  - "3.7"
-  - "nightly"
-matrix:
-  allow_failures:
-    - python: "nightly"
+jobs:
+  include:
+    - name: "18.04 pycodestyle and native run (py3.6)"
+      os: linux
+      dist: bionic
+      language: shell
+      install: sudo apt-get update; sudo apt install pycodestyle;
```

Comment:
A Travis newbie question: how Travis detects this job to run on Ubuntu and not Fedora/RHEL?

Also, as this works on Ubuntu but not Red Hat distros, will a Red Hat distro run `pycodestyle`? (but I expect `pycodestyle` will provide same output on either distro).

Reply:
Travis only supports Ubuntu natively AFAICT for os:linux, if they do add another I expect the dist: line would be how to change it.

The Travis way run another os would be to use docker services - which I plan to investigate after this commit.